### PR TITLE
feat(repo): allow setting the repo counter

### PR DESCRIPTION
**Pull request by kneal**
PR includes the following:

* Updates to repo update command to support counter field

Related to: https://github.com/go-vela/community/issues/137

Depends on: https://github.com/go-vela/types/pull/176 _should fail without this change in the mod_

### CODE REVIEW

**Review by KellyMerrick — DISMISSED** [review]
lgtm

File: action/repo_update.go [hunk 1]
```diff
@@ -124,6 +130,8 @@ EXAMPLES:
     $ {{.HelpName}} --org MyOrg --repo MyRepo --timeout 90
   4. Update a repository when config or environment variables are set.
     $ {{.HelpName}}
+	5. Update a repository with a new build number.
+		$ {{.HelpName}} --org MyOrg --repo MyRepo --counter 200
```
Comment (jbrockopp) [inline action/repo_update.go:134]: Could we get the formatting of this to be in-line with the other examples?

**Review by jbrockopp — CHANGES_REQUESTED** [review]

File: action/repo_update.go (hunk 1 above)
Comment (kneal) [reply action/repo_update.go:134]: RIP VSCode lied to me again. Will update.

Comment (kneal) [reply action/repo_update.go]: Should fix it https://github.com/go-vela/cli/pull/234/commits/a212a2d3a0865ceb1379436bec010e9d4f385e4d

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Pull request merged**